src/home/components/EditTaskForm.vue

```diff
@@ -0,0 +1,317 @@
+            />
+          </label>
+        </div>
+        <div class='task-dept input-span'>
+          <span class='input-label'>Department:</span>
+
+          <div id='demo-menu' class='mdc-menu-surface--anchor'>
+            <label
+              class='dept-select mdc-text-field mdc-text-field--outlined mdc-text-field--no-label'
+            >
+              <span class='mdc-notched-outline'>
+                <span class='mdc-notched-outline__leading'></span>
+
+                <span class='mdc-notched-outline__trailing'></span>
+              </span>
+              <input
+                @input='toggleDropdown'
+                @click='toggleDropdown'
+                v-model='dept'
+                type='text'
+                class='mdc-text-field__input'
+                placeholder='Select Department...'
+                required
+              />
+            </label>
+
+            <div
+              class='mdc-menu mdc-menu-surface'
+              v-on:MDCMenu:selected.prevent='select'
+            >
+              <ul
+                class='mdc-list'
+                role='menu'
+                aria-hidden='true'
+                aria-orientation='vertical'
+                tabindex='-1'
+              >
+                <li class='mdc-list-item' role='menuitem'>
+                  <span class='mdc-list-item__ripple'></span>
+                  <span class='mdc-list-item__text'>IT</span>
+                </li>
+                <li class='mdc-list-item' role='menuitem'>
+                  <span class='mdc-list-item__ripple'></span>
+                  <span class='mdc-list-item__text'>HR</span>
+                </li>
+                <li class='mdc-list-item' role='menuitem'>
+                  <span class='mdc-list-item__ripple'></span>
+                  <span class='mdc-list-item__text'>New Hire Paperwork</span>
+                </li>
+                <li class='mdc-list-item' role='menuitem'>
+                  <span class='mdc-list-item__ripple'></span>
+                  <span class='mdc-list-item__text'>Culture Orientation</span>
+                </li>
+                <li class='mdc-list-item' role='menuitem'>
+                  <span class='mdc-list-item__ripple'></span>
+                  <span class='mdc-list-item__text'>Other</span>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+
+        <button
+          class='edit-task-button mdc-button mdc-button--unelevated'
+          type='submit'
+          :disabled='submitButtonDisabled'
+        >
+          <span class='mdc-button__ripple'></span>
+          <span class='mdc-button__label'>Save</span>
+        </button>
+      </form>
+    </div>
+  </div>
+</template>
+
+<script>
+  import { MDCTextField } from '@material/textfield';
+  import { MDCMenu } from '@material/menu';
+
+  export default {
+    props: {
+      id: {
+        type: String,
+        required: true,
+      },
+      currentDept: {
```

Comment:
Props can also be defined in Alphabetical order like `currentDept` can move up then `id`.